components/GlobalStyle.js

```diff
@@ -29,7 +128,7 @@ const GlobalStyle = createGlobalStyle`
       }
   
       body {
-          font-family: 'Noto Sans', sans-serif;
+          font-family: 'Inconsolata', sans-serif;
```

Comment:
Because this font is a mono font, you could probably default to: 

```suggestion
          font-family: 'Inconsolata', monospace;
```